ui/lib/open-api-explorer/index.js

```diff
@@ -7,11 +7,13 @@ const EngineAddon = require('ember-engines/lib/engine-addon');
 module.exports = EngineAddon.extend({
   name: 'open-api-explorer',
 
+  babel: {
+    plugins: [require.resolve('ember-auto-import/babel-plugin')],
+  },
   included() {
     this._super.included && this._super.included.apply(this, arguments);
     // we want to lazy load these deps, importing them here will result in them being added to the
     // engine-vendor files that will be lazy loaded with the engine
```

Comment:
Is this comment now incorrect and if believed would lead to the same kind of thing happening again?

Reply:
Whoops yes, I need to remove that - the CSS bits work fine (as I’d expect because there’s no dependency tracing there, it’s just shuffling files around) - I’ll change this to warn about including the JS file

Reply:
Updated in https://github.com/hashicorp/vault/pull/7067/commits/a2d699c4738db26c988f83495b2126f241cac1b5 to be specifically about CSS and to warn about the JS and what we’re doing as a work around. Thanks!